_dashboards/maps-dashboard.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Adding Maps to your Dashboard
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps
```

Comment:
This feature isn't related to coordinate and region maps, I think it belongs to `_dashboards/maps-plugin.md`